README.md

```diff
@@ -37,15 +37,26 @@ dependencies {
 
 ##Building this project
 
-Use a recent version of [Apache Maven](http://maven.apache.org/download.cgi) (e.g., version 3.3.3) to build this code:
+1. git clone this repository.
+ 
+1. If you have not already done so, follow the Google Genomics [getting started instructions](https://cloud.google.com/genomics/install-genomics-tools) to set up your environment
+  including [installing gcloud](https://cloud.google.com/sdk/) and running `gcloud init`.
 
-    cd utils-java
-    mvn package
+1. This project now includes code for calling the Genomics API using [gRPC](http://www.grpc.io).  To use gRPC, you'll need a version of ALPN that matches your JRE version. 
+
+  2. See the [ALPN documentation](http://www.eclipse.org/jetty/documentation/9.2.10.v20150310/alpn-chapter.html) for a table of which ALPN jar to use for your JRE version.
+  2. Then download the correct version from [here](http://mvnrepository.com/artifact/org.mortbay.jetty.alpn/alpn-boot).
+
+1. Use a recent version of [Apache Maven](http://maven.apache.org/download.cgi) (e.g., version 3.3.3) to build this code:
+```
+cd utils-java
+mvn package
+```
 
 If you wish to modify this code while also modifying a project that depends upon it (e.g., [dataflow-java](https://github.com/googlegenomics/dataflow-java)), just
 
   1. Build and install the jar to your local maven repository via `mvn install`.
-  2. Then update the pom.xml of the dependent project to temporarily depend upon the particular SNAPSHOT version of utils-java now installed in your local maven repository.  For example, make a change similar to [this](https://github.com/googlegenomics/dataflow-java/commit/17122798d579e593470c864868a46092f8795dfa) but only in your local copy of pom.xml open for editing.
+  2. Then update the `pom.xml` of the dependent project to temporarily depend upon the particular SNAPSHOT version of utils-java now installed in your local maven repository.  For example, make a change similar to [this](https://github.com/googlegenomics/dataflow-java/commit/17122798d579e593470c864868a46092f8795dfa) but only in your local copy of `pom.xml` open for editing.
```

Comment:
Can "open for editing" be trimmed from this sentence? It isn't clear to me what it adds.